mitodl/mit-open
# fix channel configuration

### What are the relevant tickets?
closes https://github.com/mitodl/mit-open/issues/742

### Description (What does it do?)
There was a bug preventing channels from being saved from the admin because django counts empty json as blank and  default value for `configuration` was empty json but `configuration`  was set as blank=False


### How can this be tested?
Go to http://localhost:8063/admin/channels/fieldchannel/. You should be able to update a channel from the admin
